src/Other/Tokenizers/Tokenizer.php

```diff
@@ -8,7 +8,7 @@ interface Tokenizer
      * Tokenize a blob of text.
      *
      * @param string $string
-     * @return string[]
+     * @return list<string>
```

Comment:
What does the list annotation mean?

Reply:
I added this in https://github.com/RubixML/RubixML/pull/114/commits/47510dc78d6e01724adbaa5431edda981869824e but to expand in some more detail:

`list<string>` is a way of enforcing that the return type will not be an associative array, rather a list of strings (a list will have numeric keys) This is important because some implementations like `NGram`, `SkipGram` work on the assumption that the key of the array will be an integer, this annotation enforces this assumption. This is why the removal of the non-essential docblocks is actually meaningful in this scenario.

Reply:
Interesting! Thanks for that. We mostly use sequential (non-associative) arrays so this annotation may be very useful across the board.

Reply:
I'm glad you think so, because this was a PR that was 'testing the water' I have a larger one stashed away 🤣

Reply:
Awesome